docs/conf.py

```diff
@@ -40,7 +48,24 @@
 # -- Options for HTML output ----------------------------------------------
 
 # We want to set the RTD theme, but not if we're on RTD.
-if os.environ.get('READTHEDOCS', '') != 'True':
+if os.environ.get('READTHEDOCS', '') == 'True':
```

Comment:
Mine too.

It is what the RTD folks [recommend][1]. Honestly I'd rather just do `os.environ.has_key('READTHEDOCS')`.

[1]: https://docs.readthedocs.org/en/latest/faq.html#how-do-i-change-behavior-for-read-the-docs